pybossa/view/fileproxy.py

```diff
@@ -92,6 +93,10 @@ def encrypted_file(store, bucket, project_id, path):
     if not signature:
         current_app.logger.exception('Project id {} no signature {}'.format(project_id, path))
         raise Forbidden('No signature')
+    size_signature = len(signature)
+    if size_signature > SIGNATURE_MAX_SIZE:
+        current_app.logger.exception('Project id {}, path {} invalid signature length {}.'.format(project_id, path, size_signature))
+        raise Forbidden('Invalid signature')
```

Comment:
this is the error msg that users will see, so maybe ignore the above comment and add `signature length {} exceeds maximum length {}.' here?

Reply:
i think having generic message for user/hacker is secure than providing exact message; similar to bad username/password message instead of mentioning whether username was incorrect or password was. Will update log message as mentioned above. Thoughts?

Reply:
I am wondering if we give a clearer response, users are more likely to correct their behavior than reach out to us? 
If the task signature is not something that users can correct themself or it may expose data to an insecure state, I agree with you that we should not give many details.

Reply:
yes, task signature is supposed to be created by backend. this is to block any malicious input passed via client...